styles/map.css

```diff
@@ -18,201 +20,253 @@ html, body, #map {
 
 ::-webkit-scrollbar-track {
     visibility: hidden;
+
     border-radius: 25px;
 }
 
 ::-webkit-scrollbar-thumb {
     background: #888;
+
     border-radius: 25px;
 }
 
 input:focus {
     outline:none;
 }
 
-.filter-menu {
+#filter-menu {
     position: fixed;
+
     top: 20%;
     left: -250px;
+
     max-height: 100vh;
     width: 250px;
+
     z-index: 500;
+
     background: #202124;
+
     border-radius: 25px;
+
     transition: 0.5s;
 }
 
-.filter-menu.active {
+#filter-menu.active {
     left: 25px;
-
 }
 
-.filter-menu .filter-menu-header {
+#filter-menu .filter-menu-header {
     position: relative;
-    max-height: 75px;
-    padding-left: 5px;
+
     padding-top: 5px;
+    padding-left: 5px;
     margin-bottom: 5px;
+
+    max-height: 75px;
 }
 
-.filter-menu .filter-menu-header .header-title {
+#filter-menu .header-title {
     display: flex;
     align-items: center;
     gap: 20px;
 }
 
-.filter-menu .filter-menu-header .header-text {
+#filter-menu .header-text {
     font-size: 20px;
     font-weight: 600;
+
     color: white;
 }
 
-.filter-menu .filter-menu-header img {
+#filter-menu img {
     width: 60px;
 }
-.filter-menu .filter-menu-header .toggle-button {
+
+#toggle-menu-open-button {
     position: absolute;
     right: -75px;
-    transform: translateY(-50%) rotate(-180deg);
     top: 50%;
+
     width: 50px;
     height: 50px;
-    border-radius: 50%;
-    font-size:20px;
-    background: #345c34;
+
     display: flex;
     align-items: center;
     justify-content: center;
+
+    transform: translateY(-50%) rotate(-180deg);
+
+    font-size:20px;
+    background: #345c34;
     color: white;
+
+    border-radius: 50%;
     cursor: pointer;
     transition: 0.3s;
 }
 
-.filter-menu.active .filter-menu-header .toggle-button {
+#filter-menu.active #toggle-menu-open-button {
     right: -25px;
```

Comment:
Muss das positioning hier sein? Wenns der beste Weg ist, go for it, das ist nur sehr unschön